src/error-page.jsx

```diff
@@ -0,0 +1,16 @@
+import { useRouteError } from "react-router-dom";
```

Comment:
I recommend to have the ErrorPage file name to be PascalCase a well. 

And to have this file in the `/routes` folder.